Lib/compositor/cmap.py

```diff
@@ -4,7 +4,16 @@
 """
 
 def extractCMAP(ttFont):
-    return ttFont["cmap"].getcmap(3, 1).cmap
+    cmap = {}
+    cmapIDs = [(3, 10), (0, 3), (3, 1)]
+    for i in range(len(cmapIDs)):
```

Comment:
Just a few comments:
- you can just loop over the tuples in the list instead of using `for i in range(len(...))` and `cmapIDs[i]`;
- you don't need to call `getcmap` twice; you can save the return value and check it is not None;
- you could use the `for ... else ...` statement; the `else` branch is executed only at the end of a loop if the `break` statement is never triggered.

```diff
- for i in range(len(cmapIDs)):
-     if ttFont["cmap"].getcmap(*cmapIDs[i]):
-         cmap = ttFont["cmap"].getcmap(*cmapIDs[i]).cmap
-         break
- if not cmap:
-     from compositor.error import CompositorError
+ for id in cmapIDs:
+     cmapSubtable = ttFont["cmap"].getcmap(*id)
+     if cmapSubtable is not None:
+         cmap = cmapSubtable.cmap
+         break
+ else:
+     from compositor.error import CompositorError
```